src/ERC1155Mappings/upgradeGateMappings.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  UpgradeRegistered, UpgradeRemoved,
+} from "../../generated/UpgradePathsV1/UpgradeGate";
+
+import {
+  ContractVersion,
+  UpgradePath,
+} from "../../generated/schema";
+import { ZoraCreator1155Impl } from "../../generated/templates/ZoraCreator1155Impl/ZoraCreator1155Impl";
+import { Bytes, store } from "@graphprotocol/graph-ts";
+
+function getContractVersion(address: Bytes): ContractVersion {
+  const contractVersion = new ContractVersion(address);
+
+  const versionString = ZoraCreator1155Impl.bind(address).contractVersion();
```

Comment:
this makes sense, but it seems like it is not failing when looking for this in the factory handler [here](https://github.com/ourzora/zora-creator-subgraph/blob/main/src/ERC1155Mappings/factoryMappings.ts#L73) so do you think we'd be ok without `try` since it doesn't fail there?